### PR TITLE
DCDO flags to remove "Tune-in Live" links for completed Hour of Code events

### PR DESCRIPTION
**Pull request by bencodeorg**
Adds two new DCDO flags that can be used to remove "Watch Now"-style buttons/links for Hour of Code pages. The flags are:
- `code_bytes_episodes_aired`: an array of the episodes that have aired so far. Episodes that have aired will no longer have a button telling people to join the live episode. We'll have to update this array every afternoon after each episode has aired.
- `show_2020_ethics_ai_panel_live_links`: a true/false flag for whether to show live links to a panel discussion about ethics in AI. The panel is over at 11:15 AM PST on December 8th, so we'll flip to false after that.

## Testing story

Tested these changes manually by setting DCDO flags in my local environment.

## Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/ai.md.erb.partial [hunk 1]
```diff
@@ -15,13 +15,18 @@ social:
   "og:video:height": ""
   "og:video:type": ""
 ---
+
+<%
+  show_tune_in_link = DCDO.get('show_2020_ethics_ai_panel_live_links', true)
```
Comment (molly-moen) [inline pegasus/sites.v3/code.org/public/ai.md.erb.partial:20]: should this variable be used below?

Comment (bencodeorg) [reply pegasus/sites.v3/code.org/public/ai.md.erb.partial:20]: Oops sorry, was switching back and forth between having a variable and just getting it twice...will fix.

File: pegasus/sites.v3/code.org/public/ai.md.erb.partial [hunk 2]
```diff
@@ -416,4 +427,9 @@ We thank Microsoft for supporting our vision and mission to ensure every child h
     </p>
   </div>
   <div style="clear: both; margin-bottom: 20px"></div>
--->
\ No newline at end of file
+<<<<<<< HEAD:pegasus/sites.v3/code.org/public/ai.md.erb.partial
```
Comment (molly-moen) [inline pegasus/sites.v3/code.org/public/ai.md.erb.partial]: delete this? 😄

Comment (bencodeorg) [reply pegasus/sites.v3/code.org/public/ai.md.erb.partial]: Gosh...tough Monday. Thanks

**Review by molly-moen — APPROVED** [review]

**Pull request merged**